packages/valory/skills/staking_abci/rounds.py

```diff
@@ -48,9 +48,18 @@ class Event(Enum):
     ROUND_TIMEOUT = "round_timeout"
     NO_MAJORITY = "no_majority"
     SERVICE_NOT_STAKED = "service_not_staked"
+    SERVICE_EVICTED = "service_evicted"
     NEXT_CHECKPOINT_NOT_REACHED_YET = "next_checkpoint_not_reached_yet"
 
 
+class StakingState(Enum):
```

Comment:
Why is this not part of the contract?

Reply:
93b7dc8f10b67df1e30ed607048627d82e73acaf